auslib/admin/views/forms.py

```diff
@@ -78,6 +80,34 @@ def coercer(value):
     return coercer
 
 
+def operator(check_version=False):
+    log.debug('starting in operator, check_version is %s' % check_version)
+
+    def _validator(form, field):
+        log.debug('starting in _validator: field.data is %s' % field.data)
+        log.debug('starting in _validator, check_version is %s' % check_version)
+        # empty input is fine
+        if field.data is None:
```

Comment:
I wrote a test for it here.https://github.com/mozilla/balrog/pull/70/files#diff-3fd9d618fb33367f3022cd28992c87e3R75 , Now I'll manually check out via the UI too